gslb/ingestion/fullsync.go

```diff
@@ -295,26 +296,31 @@ func clusterSync(ctrlList []*GSLBMemberController, gsCache *avicache.AviCache) {
 			continue
 		}
 
-		if gslbutils.GetCustomFqdnMode() {
```

Comment:
@arihantg if GetCustomFqdnMode is `false`, GSName is derived from Ingress/Route hostname. During ingestion also, we do not process hostrules if this is false.  Can you please explain what is the reason of including it inside hostrule loop instead of checking hostrules if it is true?

Reply:
we do process hostrules during ingestion if customFqdn is false. have implemented same behaviour as in ingestion. Please check in https://github.com/vmware/global-load-balancing-services-for-kubernetes/blob/master/gslb/ingestion/event_handlers.go#L818